app/src/lib/stores/helpers/infer-comparison-branch.ts

```diff
@@ -20,33 +19,40 @@ type RemotesGetter = (repository: Repository) => Promise<ReadonlyArray<IRemote>>
  * @param repository The repository the branch belongs to
  * @param branches The list of all branches for the repository
  * @param currentPullRequest The pull request to use for finding the branch
- * @param currentBranch The branch we want the parent of
  * @param getRemotes callback used to get all remotes for the current repository
- * @param comparisonCache cache used to get the number of commits ahead/behind the current branch is from another branch
  */
+
 export async function inferComparisonBranch(
   repository: Repository,
   branches: ReadonlyArray<Branch>,
   currentPullRequest: PullRequest | null,
-  currentBranch: Branch | null,
-  getRemotes: RemotesGetter,
-  comparisonCache: ComparisonCache
+  getRemotes: RemotesGetter
 ): Promise<Branch | null> {
   if (currentPullRequest !== null) {
-    return getTargetBranchOfPullRequest(branches, currentPullRequest)
+    const prBranch = getTargetBranchOfPullRequest(branches, currentPullRequest)
+    if (prBranch !== null) {
+      return prBranch
+    }
```

Comment:
we weren't doing this before!? 🙀

Reply:
Yup! But honestly I don't know if this will improve things a lot, since `prBranch` can only be `null` if the target branch of the PR is from a repository that's not a remote of the current working copy...

The only thing that we're improving is that in this scenario we'll fall back to either the upstream or the origin default branch.